sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/_utils.py

```diff
@@ -40,6 +41,26 @@ def ns_to_duration(nanoseconds):
         days, hours, minutes, seconds, microseconds
     )
 
+_INSTRUMENTATIONS_BIT_MASK = 0
+_INSTRUMENTATIONS_BIT_MASK_LOCK = threading.Lock()
+
+def get_instrumentations():
+    return _INSTRUMENTATIONS_BIT_MASK
+
+
+def add_instrumentation(instrumentation_name):
+    with _INSTRUMENTATIONS_BIT_MASK_LOCK:
+        global _INSTRUMENTATIONS_BIT_MASK  # pylint: disable=global-statement
+        instrumentation = _INSTRUMENTATIONS_BIT_MAP.get(instrumentation_name, 0)
```

Comment:
"instrumentation" would be the bit representing that instrumentation, correct? I think this variable name could be clearer.